### PR TITLE
doc: adjusted Sidewalk intro section

### PR DESCRIPTION
**Pull request by annwoj**
Removed about sidewalk rst file
Added descriptions to the main page instead

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
 
-Welcome to nRF Connect SDK - Amazon Sidewalk
-############################################
+Welcome to the nRF Connect SDK - Amazon Sidewalk
+################################################
 
-Amazon Sidewalk is a shared network that allows your devices to work better at your home and outside of it.
-This documentation provides instructions on how to correctly set up Amazon Sidewalk from the perspective of the `nRF Connect SDK`_.
+Amazon Sidewalk is a shared network designed to provide a stable and reliable connection to your devices allowing them to work better at your home and outside of it.
+Every Ring and Echo device can act as a gateway device, meaning it can share a portion of internet bandwidth providing the connection and services to Sidewalk end devices.
```
Comment (ktaborowski) [inline doc/index.rst:7]: Not every Ring and Echo device can be a Sidewalk gateway - there is a list in Amazon doc with supported devices https://docs.sidewalk.amazon/introduction/sidewalk-gateways.html

File: doc/index.rst [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
+
+.. note::
+   Amazon Sidewalk is available only in the United States of America.
+   To an extent, any Sidewalk gateway functionality might be used outside of the U.S., however, it should be used only for the Amazon Sidewalk endpoint development purposes.
+   In addition, we recommend that you consult with your local regulatory bodies, and check if the gateway is allowed to operate its radio in your locale, as U.S. license-free band devices, only for development.
+
+Amazon Sidewalk for the nRF Connect SDK is based on two variants, one using Bluetooth® LE (more suited for home applications) and the other one using sub-GHz with the Semtech SX1262 radio transceiver (for applications requiring longer range).
```
Comment (ktaborowski) [inline doc/index.rst:14]:
```suggestion
Amazon Sidewalk for the nRF Connect SDK is based on two variants, one using Bluetooth® LE (more suited for home applications) and the other one using sub-GHz with the Semtech radio transceiver (for applications requiring longer range).
```
minor: less precise, but more futureproof

File: doc/index.rst [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
+The latter also expands to two options: Sidewalk over LoRa and Sidewalk over FSK (having a shorter range but with higher throughput compared to LoRa).
+
+Amazon Sidewalk products can be tested and developed using the `nRF52840`_ SoC and the `nRF Connect SDK`_.
```
Comment (ktaborowski) [inline doc/index.rst:17]:
```suggestion
Amazon Sidewalk products can be tested and developed using the nRF SoC together with the `nRF Connect SDK`_.
```
minor: Same as above

**Review by ktaborowski — CHANGES_REQUESTED** [review]
Please fix gateway device information. Overall looks good

File: doc/index.rst (hunk 1 above)
Comment (annwoj) [reply doc/index.rst]: added the reference

**Review by ktaborowski — APPROVED** [review]

**Pull request merged**